staging/kos/pkg/networkfabric/ovs/doc.go

```diff
@@ -0,0 +1,94 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package ovs defines a network fabric that implements k8s.io/examples/staging/kos/pkg/networkfabric.Contract .
+//
+// The fabric is thread-safe.
+//
+// The fabric is based on an Open vSwitch (OvS) bridge, which gets configured
+// through OpenFlow.
+//
+//******************************************************************************
+// Currently, THE FABRIC DOES NOT WORK IF ANOTHER OvS BRIDGE EXISTS ON THE NODE.
```

Comment:
This is surprisingly fragile.  Couldn't we remove this restriction by using a bridge with a fixed name?

Reply:
The bridge already has a fixed name. The problem is, when listing existing local network interfaces, you need to query all the existing interfaces connected to that bridge, and IIRC when I wrote that code the OvS CLI did not have a straightforward way to do this. So what is done in the code is listing all the network interfaces managed by the OvS daemon, that is, all the network interfaces of all the OvS bridges on the node. I'm sure there is a way to get the behavior we want, but the OvS CLI doesn't make our life easy in that regard. Will dig more into this ASAP (probably later today).

Reply:
Actually, I did some research and I think there's a reasonably simple way to remove the fragility. Will get to it later today.